packages/ai/src/generate-text/stream-text-onfinish-abort.test.ts

```diff
@@ -0,0 +1,224 @@
+import { convertArrayToReadableStream } from '@ai-sdk/provider-utils/test';
+import { describe, expect, it, vi } from 'vitest';
+import { MockLanguageModelV2 } from '../test/mock-language-model-v2';
+import { streamText } from './stream-text';
+
+describe('streamText onFinish with abort', () => {
```

Comment:
can you integrate this test into stream-text test and in particular into the ui message stream section (since it is specifically for toUIMessageStream)

Reply:
moved to existing file